app/gui/src/project-view/components/visualizations/JSONVisualization/JsonValueWidget.vue

```diff
@@ -10,8 +10,13 @@ const emit = defineEmits<{
 </script>
 
 <template>
+  <span
```

Comment:
It looks like this can occur anywhere inside a JSON structure--in that case, it would be nice to have some styling that can help distinguish this from any surrounding JSON data.